include/meson.build

```diff
@@ -0,0 +1,18 @@
+# include build file
+#========================================================================
+
+include_dir = meson.current_source_dir()
+proj_inc += include_directories('.')
+
+# include directories
+#========================================================================
+
+
+include_dirs = [
+    'config_h',
+    'macros']
+
+
+foreach dir : include_dirs
+    subdir(dir)
+endforeach
```

Comment:
You are being inconsistent here.
You feel comfortable to use `foreach` here instead of 

```
subdir('config_h')
subdir('macros')
```

But you don't like to do the same in the examples above.

Reply:
its a matter of clarity; i worry that the larger, more complicated for loops could make things look messy